File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-04/plagiarized/L6/05/pengulangan.kt
package IR

internal object inputoutput {
    fun hitHasil(a: Int) {
        var a = a
        while (a <= 10) {
            println(a.toString() + "\t\t" + a * 1.609)
            a++
        }
    }

    @JvmStatic
    fun main(args: Array<String>) {
        // print seusatu
        print("Miles\t\tKilometers\n")
        print("-------------------------------\n")
        // pake pengulangan while
        hitHasil(1)
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-04/plagiarized/L4/05/pengulangan.ts
import { forEach } from 'lodash';

class inputoutput {
  static hitHasil(m: number): void {
    while (m <= 10) {
      console.log(`${m}\t\t${m * 1.609}`);
      m++;
    }
  }

  public static main(args: string[]): void {
    let m = 1;
    console.log('Miles\t\tKilometers');
    console.log('-------------------------------');
    this.hitHasil(m);
  }
}

inputoutput.main(process.argv.slice(2));

